src/common/components/formikWrappers/FormikDropdown.tsx

```diff
@@ -17,23 +42,38 @@ export type HdsOptionType = {
   [key: string]: any;
 };
 
-function FormikDropdown(props: Props) {
-  // HDS Dropdown expects default value to be an object. Find correct option object from array.
-  const getSelectDefault = (options: OptionType[], value?: string) => {
-    return options.find((option: OptionType) => option.value === value);
+function FormikDropdown({
+  name,
+  id,
+  onChange = () => ({}),
+  value,
+  helperText,
+  labelText,
+  options,
+  ...rest
+}: Props) {
+  const { t } = useTranslation();
+  const [field, meta, helpers] = useField(name);
+  const [isInvalid, setIsInvalid] = useState(false); // Lol where do we call this
```

Comment:
The form is validated against the yup schema or the validate function. I don't think that you need to call any error logic on this level by default. The `meta` object actually contains information about the error state.

https://formik.org/docs/api/useField#fieldmetapropsvalue

Reply:
That makes sense. I did it like this for two reasons, first due to typing errors - second as a first step in trying to fake onBlur / visited for these dropdowns.

Reply:
For doing those things, `helpers` may be useful. That object contains for instance `setTouched` which is what you'd likely end up calling if you "faked" onBlur.

Reply:
Found a much simpler way to handle the "faked" onBlur!